services/app/app/__main__.py

```diff
@@ -82,4 +82,24 @@ def weather_get(payload: dict, metadata: dict) -> Optional[Tuple[int, Union[dict
     return 200, weather.to_dict()
 
 
+@server.route('Inventory.get')
+def inventory_get(payload: dict, metadata: dict) -> Optional[Tuple[int, Union[dict, list]]]:
+    inventory = db.inventory_get(
```

Comment:
It only looks like the happy path (`200`) is implemented by this function. I think there were some more error codes in the spec. The case where the inventory is not found (`404`) and likely some permission error if the inventory belongs to another player.